ddl/ddl_worker.go

```diff
@@ -214,10 +220,12 @@ func (d *ddl) handleDDLJobQueue() error {
 			schemaVer = d.runDDLJob(t, job)
 			if job.IsFinished() {
 				binloginfo.SetDDLBinlog(d.workerVars.BinlogClient, txn, job.ID, job.Query)
```

Comment:
@tiancaiamao Cancel is the old logic.  https://github.com/pingcap/tidb/pull/3498/files#diff-c87d71f8121c3ee6f07826da99de9eabR189